packages/sdk/src/controllers/SubscriberController.ts

```diff
@@ -66,22 +66,47 @@ export class SubscriberController {
         callBack && callBack(unit as MeasurementUnit);
     }
 
+
     /**
-     * Listener on the state of the currently selected frame, if this changes, this listener will get triggered with the updates
-     * @param frameLayout Stringified object of FrameLayoutType
+     * @deprecated use `onSelectedFramesLayoutChanged` instead
+     * Listener on the state of the currently selected frames
+     * Whenever multiple frames are selected, it always will be the first selected `FrameLayoutType`
+     * @param frameLayout Stringified array of `FrameLayoutType` objects
      */
     onSelectedFrameLayoutChanged = (frameLayout: string) => {
-        const callBack = this.config.onSelectedFrameLayoutChanged;
-        callBack && callBack(JSON.parse(frameLayout));
+        const callBack = this.config.onSelectedFramesLayoutChanged;
+        const frames = JSON.parse(frameLayout);
+        callBack && callBack(frames[0]);
+    };
+
+    /**
+     * Listener on the state of the currently selected frames, if this changes, this listener will get triggered with the updates
+     * @param framesLayout Stringified array of FrameLayoutType objects
+     */
+    onSelectedFramesLayoutChanged = (framesLayout: string) => {
+        const callBack = this.config.onSelectedFramesLayoutChanged;
+        callBack && callBack(JSON.parse(framesLayout));
     };
 
     /**
+     * @deprecated use `onSelectedFramesContentChanged` instead
```

Comment:
same here, call works and uses the newest endpoint so I wouldn't deprecate it

Reply:
same answer as above